runelite-client/src/main/java/net/runelite/client/plugins/reportbutton/ReportButtonPlugin.java

```diff
@@ -110,6 +154,12 @@ public void updateReportButtonTime()
 			return;
 		}
 
+		if (config.raidTimer() && raidStartTime != null)
```

Comment:
Maybe add the RAID to the enum of choices instead of adding boolean?

Reply:
then users couldn't use any other report button option while not in raid.